frontend/test_runner.go

```diff
@@ -60,16 +62,20 @@ func RunTests(ctx context.Context, client gwclient.Client, spec *dalec.Spec, ref
 		return err
 	}
 
-	type testPair struct {
-		st     llb.State
-		t      *dalec.TestSpec
-		stdios map[int]llb.State
-		opts   []llb.ConstraintsOpt
+	ctrWithDeps := ctr.With(withTestDeps)
+
+	fSt, err := client.(frontendClient).CurrentFrontend()
```

Comment:
Should we use [GetCurrentFrontend(client)](https://github.com/Azure/dalec/blob/f604f38ff1bd447624e1e04aa39337f6f9ab64b7/frontend/build.go#L219) instead?